test/integration/node-specific/mongo_client.test.ts

```diff
@@ -8,16 +8,15 @@ import {
   type CommandFailedEvent,
```

Comment:
Two tests related to the `appName` in this file broke with the changes in this PR.  I decided to just refactor this file to async-await while I was working in it.